be/src/storage/lake/tablet_manager.cpp

```diff
@@ -254,16 +274,13 @@ StatusOr<TabletMetadataPtr> TabletManager::get_tablet_metadata(const string& pat
 Status TabletManager::delete_tablet_metadata(int64_t tablet_id, int64_t version) {
     auto location = tablet_metadata_location(tablet_id, version);
     _metacache->erase(location);
-    return fs::delete_file(location);
+    return ignore_not_found(fs::delete_file(location));
```

Comment:
should only ignore not found if the version == 1?

Reply:
done